cmd/syntactic-code-intel-worker/shared/indexing_worker.go

```diff
@@ -0,0 +1,39 @@
+package shared
+
+import (
+	"context"
+	"github.com/sourcegraph/log"
+	"time"
+
+	"github.com/sourcegraph/sourcegraph/internal/observation"
+	"github.com/sourcegraph/sourcegraph/internal/workerutil"
+	"github.com/sourcegraph/sourcegraph/internal/workerutil/dbworker"
+	dbworkerstore "github.com/sourcegraph/sourcegraph/internal/workerutil/dbworker/store"
+)
+
+func NewIndexingWorker(ctx context.Context, observationCtx *observation.Context, workerStore dbworkerstore.Store[*SyntacticIndexRecord], config IndexingWorkerConfig) *workerutil.Worker[*SyntacticIndexRecord] {
+
+	name := "syntactic_code_intel_indexer"
```

Comment:
```suggestion
	name := "syntactic_code_intel_indexing_worker"
```

To match the existing naming conventions for other workers.